demo/vl-search-results.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+	<head>
+        <title>vl-search-results voorbeeld</title>
+        <script type="module" src="/node_modules/vl-ui-demo/dist/vl-demo-all.js"></script>
+
+        <script type="module" src="/src/vl-search-results.js"></script>
+        <script type="module" src="/src/vl-search-result.js"></script>
+
+        <link rel="stylesheet" type="text/css" href="/src/style.css" />
+    </head>
+
+	<body>
+        <vl-demo-page data-vl-webcomponent="vl-search-results" data-vl-link="https://overheid.vlaanderen.be/webuniversum/v3/documentation/components/vl-ui-search-results">
+            <vl-demo data-vl-title="Search results">
+                <vl-search-results>
+                    <vl-search-result>
+                        <a href="#" slot="title">Vlaanderenkiest.be</a>
+                        <time slot="sub-title">Maandag 22 oktober 2018</time>
+                        <dl slot="content">
+                            <dt>Vlaanderenkiest.be</dt>
+                            <dd>Verkiezingsresultaten op Vlaanderenkiest.be...</dd>
+                            <dt>Vlaanderen intern</dt>
+                            <dd>Werkt u bij de Vlaamse overheid...</dd>
+                        </dl>
```

Comment:
Wat als we een "rij" willen toevoegen zonder de ":"? M.a.w. enkel een <dt/> of <dd/> zonder de andere? (Momenteel werd er een ":" gerenderd als je enkel <dt> zou toevoegen zonder de <dd>)

Reply:
Momenteel heb ik alleen list content voorzien omdat volgens de mockup alleen daar nood aan was, maar het is ook mogelijk om gewone (niet `<dl>`) content slots te voorzien.